mungegithub/github/github.go

```diff
@@ -539,6 +540,26 @@ func (obj *MungeObject) RemoveLabel(label string) error {
 	return nil
 }
 
+// Priority returns the priority an issue was labeled with.
+// The labels must take the form 'priority/P?[0-9]+'
+// or math.MaxInt32 if unset
+func (obj *MungeObject) Priority() int {
+	priority := math.MaxInt32
```

Comment:
I was thinking whether a default priority such as p2 makes sense. This spares most people from applying priority labels explicitly.